spark/src/main/scala/io/spark/ddf/etl/TransformationHandler.scala

```diff
@@ -96,26 +101,20 @@ class TransformationHandler(mDDF: DDF) extends CoreTransformationHandler(mDDF) {
     val rMapped = dfrdd.map {
```

Comment:
@dabaitu what likely explains this is as follows: one or more of the `rsess.eval()` can throw/threw an exception, which caused a `Nothing` to be returned to the Scala side, which then causes the entire `RDD` to be `RDD[Nothing]`. By expressly specifying `REXP` you ensure that the `RDD` is `RDD[REXP]`.